### PR TITLE
PLAT-140745: Refine logic to handle the last input type

### PR DESCRIPTION
**Pull request by 0x64**
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
The current API to handle the last input type exported by callback prop for `SpotlightRootDecorator` and it will be better to be exported in a normal way.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated to export key APIs from the module, not through callback function prop.
Also, for more safe recognition of input type, separate catching events for mouse input and touch input.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-140745

### CODE REVIEW

File: packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js [hunk 1]
```diff
@@ -32,17 +61,7 @@ const defaultConfig = {
...
-
-	/**
-	 * Specifies the id of the React DOM tree root node
-	 *
-	 * @type {String}
-	 * @default 'root'
-	 * @public
-	 * @memberof spotlight/SpotlightRootDecorator.SpotlightRootDecorator.defaultConfig
-	 */
-	rootId: 'root'
```
Comment (jeonghee27) [inline packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js:45]: :cry:

**Review by jeonghee27 — APPROVED** [review]
LGTM

File: packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js [hunk 2]
```diff
@@ -191,5 +195,8 @@ const SpotlightRootDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 export default SpotlightRootDecorator;
 export {
-	SpotlightRootDecorator
+	SpotlightRootDecorator,
+	activateInputType,
+	getInputType,
+	setInputType
```
Comment (MikyungKim) [inline packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js:201]: Please add @export these from line 7.

Comment (MikyungKim) [reply packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js:201]: Since these are private methods, let's not add.

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**